test/internal/modules/charge-information/forms/charge-categories/volume.js

```diff
@@ -115,10 +115,10 @@ experiment('internal/modules/charge-information/forms/charge-category/volume', (
         expect(result.error).to.exist();
       });
 
-      test('must not be zero', async () => {
+      test('must be greater than 0', async () => {
```

Comment:
just a text change based on the ticket: 

Numeric values equal to or greater than 0

Reply:
Fixed